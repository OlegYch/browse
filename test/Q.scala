case class B(xyz: Int)
trait Q {
	def m(i: Int) = i
	def m2(a: Int, b: Int) = a + b
	def self = this
}

