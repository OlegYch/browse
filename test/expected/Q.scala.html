<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
    <head>
        <meta http-equiv="Content-Type" content="text/html;charset=utf-8" ></meta>
        <meta http-equiv="Expires" content="0" />
        <title>Q.scala</title>
        <script type="text/javascript" src="jquery-all.js"></script>
        <script type="text/javascript" src="linked.js"></script>
        <link rel="stylesheet" type="text/css" href="style.css" title="Style"></link>
    </head>
    <body>
        <pre>
<span class="keyword">case class</span> <a title="class B extends AnyRef with Product with Serializable" id="17256">B</a><a href="#17256" title="Product" class="delimiter">(</a><a title="Int" id="7727">xyz</a>: <span title="Int">Int</span><span class="delimiter">)</span>
<span class="keyword">trait</span> <a title="trait Q extends AnyRef" id="7137">Q</a> <span title="Unit" class="delimiter">{</span>
	<span class="keyword">def</span> <a title="(i: Int)Int" id="16512">m</a><span class="delimiter">(</span><a title="Int" id="16521">i</a>: <span title="Int">Int</span><span class="delimiter">)</span> = <a href="#16521" title="Int">i</a>
	<span class="keyword">def</span> <a title="(a: Int, b: Int)Int" id="16513">m2</a><span class="delimiter">(</span><a title="Int" id="16515">a</a>: <span title="Int">Int</span>, <a title="Int" id="16516">b</a>: <span title="Int">Int</span><span class="delimiter">)</span> = <a href="#16515" title="Int">a</a> <span title="(x: Int)Int">+</span> <a href="#16516" title="Int">b</a>
	<span class="keyword">def</span> <a title="=&gt; Q" id="16514">self</a> = <a href="#7137" title="Q" class="keyword">this</a>
<span class="delimiter">}</span>


        </pre>
    </body>
</html>
